--- a/src/vpux_driver_compiler/CMakeLists.txt	2026-05-27 11:31:48.458085196 +0200
+++ b/src/vpux_driver_compiler/CMakeLists.txt	2026-05-27 11:31:48.463124436 +0200
@@ -19,7 +19,7 @@
 
 if(BUILD_COMPILER_FOR_DRIVER)
     set(INSTALL_COMPONENT ${CID_COMPONENT})
-    set(INSTALL_DESTINATION cid)
+    set(INSTALL_DESTINATION ${CMAKE_INSTALL_PREFIX})
 else()
     set(INSTALL_COMPONENT ${CIP_COMPONENT})
     set(INSTALL_DESTINATION cip)
@@ -27,6 +27,8 @@
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
 
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../vpux_compiler/include)
+
 add_subdirectory(src)
 add_subdirectory(test)
 
--- a/src/vpux_driver_compiler/cmake/install_target_with_debug_files.cmake	2026-05-27 11:31:48.458760847 +0200
+++ b/src/vpux_driver_compiler/cmake/install_target_with_debug_files.cmake	2026-05-27 11:31:48.463760814 +0200
@@ -30,7 +30,7 @@
         endif()
     else()
         install(TARGETS ${target}
-            LIBRARY DESTINATION ${destination} COMPONENT ${component}
+            LIBRARY DESTINATION lib64 COMPONENT ${component}
             RUNTIME DESTINATION ${destination} COMPONENT ${component})
     endif()
 endfunction()
--- a/src/vpux_driver_compiler/src/vpux_compiler_l0/CMakeLists.txt	2026-05-27 11:31:48.459760841 +0200
+++ b/src/vpux_driver_compiler/src/vpux_compiler_l0/CMakeLists.txt	2026-05-27 11:31:48.466256471 +0200
@@ -69,7 +69,7 @@
 install(
         FILES
             "${PROJECT_SOURCE_DIR}/src/vpux_driver_compiler/include/npu_driver_compiler.h"
-        DESTINATION ${INSTALL_DESTINATION}
+        DESTINATION include
         COMPONENT ${INSTALL_COMPONENT})
 
 # Install the openvino_intel_npu_compiler library to the same path of mlir_compiler
