--- a/src/vpux_driver_compiler/CMakeLists.txt
+++ b/src/vpux_driver_compiler/CMakeLists.txt
@@ -31,14 +31,3 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)

 add_subdirectory(src)
 add_subdirectory(test)
-
-install(
-    FILES
-        "${PROJECT_SOURCE_DIR}/src/vpux_driver_compiler/CHANGES.txt"
-    DESTINATION ${INSTALL_DESTINATION}
-    COMPONENT ${INSTALL_COMPONENT})
-
-install(
-    FILES
-        "${PROJECT_SOURCE_DIR}/src/vpux_driver_compiler/README.md"
-    DESTINATION ${INSTALL_DESTINATION}
-    COMPONENT ${INSTALL_COMPONENT})
