diff --git a/CMakeLists.txt b/CMakeLists.txt
index 044996e..c7e618e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -415,9 +415,11 @@ if(ONNX_USE_LITE_PROTO)
   endif()
 endif()
 add_library(onnx_proto ${ONNX_PROTO_SRCS})
+set_target_properties(onnx_proto PROPERTIES VERSION ${ONNX_VERSION} SOVERSION ${ONNX_VERSION})
 add_onnx_global_defines(onnx_proto)
 
 add_library(onnx ${ONNX_SRCS} ${ONNX_PROTO_SRCS})
+set_target_properties(onnx PROPERTIES VERSION ${ONNX_VERSION} SOVERSION ${ONNX_VERSION})
 add_dependencies(onnx onnx_proto)
 set_source_files_properties(${ONNX_SRCS} PROPERTIES CXX_VISIBILITY_PRESET hidden)
 add_onnx_global_defines(onnx)
